go.mod

```diff
@@ -14,6 +14,7 @@ require (
 	github.com/google/pprof v0.0.0-20211122183932-1daafda22083
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
 	github.com/opencontainers/runtime-spec v1.0.2
+	github.com/pkg/errors v0.9.1
```

Comment:
Are we using anything specifically from this package or can we just use `"errors"`?

Reply:
Good call, I was using it for `errors.Wrap` but doesn't seem worth importing it for that. Removed.